Presentation.Web/Tests/it-system/regularUser.System.e2e.spec.ts

```diff
@@ -16,31 +16,27 @@ describe("Regular user tests", () => {
         browser.waitForAngular();
     });
 
-    it("Use filter is disabled", () => {
+    it("Apply and delete filter buttons are disabled", () => {
```

Comment:
Wording? Better: it("should have the apply and delete filter buttons disabled"..